versioned_docs/version-latest/protocol/wallets/subwallet.md

```diff
@@ -72,9 +72,9 @@ This tutorial assumes you have already completed the setup for SubWallet. If you
 
 :::
 
-SubWallet supports adding custom networks. This can be helpful for in-development networks such as the Subspace Network as we have regularly changing RPC points and versions of testnets, and soon various domains. Below you will find a simple guide on how to add these new networks. Additionally we will try and keep an updated list of active RPC points below for development. 
+SubWallet supports adding custom networks. This can be helpful for in-development networks such as the Subspace Network as we have regularly changing RPC endpoints and versions of testnets, and soon various domains. Below you will find a simple guide on how to add these new networks. Additionally we will try and keep an updated list of active RPC endpoints below for development. 
 
-:::info RPC Endpoints
+:::info RPC Endendpoints
```

Comment:
Typo.

Reply:
Guess I was a little too quick on this PR, issue resolved here: https://github.com/subspace/subspace-docs/pull/196